<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hasil_sniffing</class>
 <widget class="QMainWindow" name="Hasil_sniffing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hasil Sniffing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>591</width>
      <height>361</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>No</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hasil</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Waktu</string>
     </property>
    </column>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>177</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Kemballi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>331</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Hasil Sniffing Nama</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
